app/src/test/java/io/intrepid/skeleton/testutils/TestFileUtils.java

```diff
@@ -0,0 +1,49 @@
+package io.intrepid.skeleton.testutils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+public class TestFileUtils {
```

Comment:
Ditto this perhaps being factored out into a library rather than repeated across all template projects.